metrics_api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -16,7 +16,7 @@ class Meter
       UP_DOWN_COUNTER = Instrument::UpDownCounter.new
       OBSERVABLE_UP_DOWN_COUNTER = Instrument::ObservableUpDownCounter.new
 
-      NAME_REGEX = /\A[a-zA-Z][-.\w]{0,62}\z/
+      NAME_REGEX = %r{\A[a-zA-Z][-./\w]{0,254}\z}
```

Comment:
Nice adjustment! I always forget about `%r`.

Reply:
Thank you!! Rubocop reminded me about it, but I didn't have it very present either lol